js/components/discussions/threads.vue

```diff
@@ -1,21 +1,19 @@
 <template>
 <div class="list-group resources-list smaller">
-    <discussion-thread v-for="discussion in discussions" :discussion="discussion">
+    <discussion-thread v-ref:threads v-for="discussion in discussions" :discussion="discussion" track-by="id">
     </discussion-thread>
-    <a class="list-group-item add new-discussion"
-        @click="displayForm" v-show="formDisplayed">
+    <a class="list-group-item add new-discussion" @click="displayForm" v-show="!formDisplayed">
         <div class="format-label pull-left">+</div>
-        <h4 class="list-group-item-heading">
-            {{ _('Start a new discussion') }}
-        </h4>
+        <h4 class="list-group-item-heading">{{ _('Start a new discussion') }}</h4>
     </a>
-    <div class="list-group-item list-group-form list-group-form-discussion animated"
-        v-show="formDisplayed" v-el:form>
+    <div v-el:form id="discussion-create" v-show="formDisplayed" v-if="currentUser"
```

Comment:
Yes, but we should think/work on how we display and make it the same on every component because this is something new.
Until now, no component works this way, we never announce it needs login, we display on the login screen the reason that triggers it.

Right now, I just changed the moment where the auth is triggered.
Using YouCKAN, the login screen displays:
- you need to be authentified to post a comment
- you need to be authentified to create a discussion